application.example.yaml

```diff
@@ -92,5 +94,5 @@ threepid.medium.email.connectors.smtp.login: "matrix-identity@example.org"
 # Password for the account
 threepid.medium.email.connectors.smtp.password: "ThePassword"
 
-# The e-mail to send as. If empty, will be the same as login
+# The e-mail to send as.
```

Comment:
What we discussed was that `empty` was misleading, as what we really meant was `not set`.
Typically, this value would be set as the login and email would be different.
Removing that part doesn't make it more useful I think.

Reply:
Given that the ```smtp.login``` may be not an email address but just an username, I think it is error-prone. Unless we implement a dirty mechanism to verify that ```identity.from``` is a correct email address. Also, the ```e-mail to send as``` is usuallly a "no-reply" address, and should not be the same as the username login.

Reply:
makes sense, fine by me.